tester_app/client/assets/stylesheets/_bootstrap-variables-customization.scss

```diff
@@ -0,0 +1,18 @@
+// This file is used as part of the bootstrap-sass-loader customization
+// Included from bootstrap-sass.config.js
```

Comment:
@justin808 this *is* included by Rails, see https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/app/assets/stylesheets/_bootstrap-custom.scss#L5